### PR TITLE
Release notes for 1.9

### PR DESCRIPTION
**Pull request by brson**

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -1,3 +1,291 @@
...
+  fields and methods cannot prevent the proper use of public fields
+  and methods][1.9fv].
+* [The parser considers unicode codepoints in the
+  `PATTERN_WHITE_SPACE` category to be whitespace][1.9ws].
+
+Stabilized APIs
+---------------
+
+* [`std::panic`]
+* [`std::panic::catch_unwind`][] (renamed from `recover`)
```
Comment (sfackler) [inline RELEASES.md]: Bad links?

File: RELEASES.md [hunk 2]
```diff
@@ -1,3 +1,291 @@
...
+
+Libraries
+---------
+
+* [`std::sync::Once` is poisoned if its initialization function
+  fails][1.9o].
+* [`cell::Ref` and `cell::RefMut` can contain unsized types][1.9cu].
+* [Most types implement `fmt::Debug`][1.9db].
+* [The default buffer size used by `BufReader` and `BufWriter` was
+  reduced to 8K, from 64K][1.9db]. This is inline with the buffer size
```
Comment (sfackler) [inline RELEASES.md]: "in line"

File: RELEASES.md [hunk 3]
```diff
@@ -1,3 +1,291 @@
...
+* [`UdpSocket::set_nonblocking`]
+
+Libraries
+---------
+
+* [`std::sync::Once` is poisoned if its initialization function
+  fails][1.9o].
+* [`cell::Ref` and `cell::RefMut` can contain unsized types][1.9cu].
+* [Most types implement `fmt::Debug`][1.9db].
+* [The default buffer size used by `BufReader` and `BufWriter` was
```
Comment (sfackler) [inline RELEASES.md]: It's also used in some other places like `io::copy`. Not sure if it's worth mentioning, though.

**Comment by jethrogb** [comment]
"Most types implement fmt::Debug." links to wrong PR

File: RELEASES.md [hunk 4]
```diff
@@ -1,3 +1,291 @@
...
+* [`UdpSocket::recv`]
+* [`UdpSocket::set_nonblocking`]
+
+Libraries
+---------
+
+* [`std::sync::Once` is poisoned if its initialization function
+  fails][1.9o].
+* [`cell::Ref` and `cell::RefMut` can contain unsized types][1.9cu].
+* [Most types implement `fmt::Debug`][1.9db].
```
Comment (shengsheng) [inline RELEASES.md]: Bad link.

File: RELEASES.md (hunk 2 above)
Comment (durka) [inline RELEASES.md]: also duplicate link target `1.9db`

File: RELEASES.md (hunk 1 above)
Comment (brson) [inline RELEASES.md]: These renames didn't actually exist on master yet when I was writing the notes, and still haven't been published in a nightly.

**Comment by alexcrichton** [comment]
r=me when this is ready to go

**Comment by brson** [comment]
Thanks for the reviews.

**Pull request merged**